docs/Build.md

```diff
@@ -0,0 +1,72 @@
+Naggum Build Guide
+==================
+
+Windows
+-------
+
+To build Naggum under Windows, just use Visual Studio or MSBuild like that:
+
+    $ cd naggum
+    $ nuget restore
+    $ msbuild /p:Configuration=Release Naggum.sln
+
+Linux
+-----
+
+See general build instructions for Linux in the file `.travis.yml` inside of the
+Naggum root source directory.
+
+You'll need [Mono][mono], [NuGet][nuget] and [F# Compiler][fsharp] installed in
+you current environment. Some of them may or may not be part of your Mono
+installation; just make sure you have them all properly installed.
+
+Below is an example of setting up these tools on [NixOS Linux][nixos]; feel free
+to add instructions for any other non-standard distributives.
```

Comment:
Yeah, silly typo.

Well, here I just trying to claim that the "standard" distribution is the one in that you can just install Mono and FSharp, and it'll work out of the box. Non-standard is the one that requires additional guide.

I think I'll replace this with "additional distributions".